package/kedro_viz/models/experiment_tracking.py

```diff
@@ -111,3 +114,9 @@ def load_tracking_data(self, run_id: str):
 
 def get_dataset_type(dataset: AbstractVersionedDataSet) -> str:
     return f"{dataset.__class__.__module__}.{dataset.__class__.__qualname__}"
+
+
+def get_dataset_module_class(dataset: AbstractVersionedDataSet) -> str:
+    class_name = f"{dataset.__class__.__qualname__}"
+    _, dataset_type, dataset_file = f"{dataset.__class__.__module__}".rsplit(".", 2)
+    return f"{dataset_type}.{dataset_file}.{class_name}"
```

Comment:
The naming is kind of confusing given we already call the "full path" `dataset_type` (which is probably already confusing, but it's also in the flowchart part of kedro-viz source, so probably let's leave it that way for now).
```suggestion
def get_abbreviated_dataset_type(dataset: AbstractVersionedDataSet) -> str:
    """e.g. plotly.plotly_dataset.PlotlyDataSet"""
    abbreviated_module_name = ".".join(dataset.__class__.__module__.split(".")[-2:])
    return f"{abbreviated_module_name}.{dataset.__class__.__qualname__}"
```